backend/src/models/picker_options.js

```diff
@@ -208,6 +208,26 @@ const tableLookupValues = (projectId, contractId) => {
       label: `project_deliverable.deliverable_name`,
       queryAdditions: `WHERE project_deliverable.deliverable_name IS NOT NULL`,
     },
+    {
+      id: "contractresource",
+      name: "contract_resource",
+      title: "Contract Resource",
+      description: "",
+      table: "data.contract_resource",
+      value: "contract_resource.id",
+      label: `(r.resource_last_name || ', ' || r.resource_first_name)`,
```

Comment:
does this work, as how does it know '**r**' is the contract_resource table, this is not declared anywheres. -- ahh i see its in the join (queryAdditions)